From: Paul Boddie <paul@boddie.org.uk>
Date: Tue, 15 Aug 2023 23:50:33 +0200
Subject: Fix Python packaging metadata.

The Python packaging name for pdfminer may leak out into requires.txt
files used by the Python import system, causing Python to refuse to
look for the installed module/package despite it being present and
readily discoverable. Various XStatic packages appear to have
currently unavailable versions for Debian.

moin (2.0+git20230330.c55318e3-1) unstable; urgency=medium

  * Initial release.
---
 setup.py | 9 +++------
 1 file changed, 3 insertions(+), 6 deletions(-)

diff --git a/setup.py b/setup.py
index 8f640da..a095656 100644
--- a/setup.py
+++ b/setup.py
@@ -84,20 +84,17 @@ setup_args = dict(
         'pygments>=1.4',  # src code / text file highlighting
         'Werkzeug<2.4.0',  # wsgi toolkit
         'whoosh>=2.7.0',  # needed for indexed search
-        'pdfminer3',  # pdf -> text/plain conversion
+        'pdfminer.six',  # pdf -> text/plain conversion
         'passlib>=1.6.0',  # strong password hashing (1.6 needed for consteq)
         'setuptools>=51',  # dependency with setuptools_scm
         'sqlalchemy<2.0',  # used by sqla store
         'XStatic>=0.0.2',  # support for static file pypi packages
-        'XStatic-Bootstrap==3.1.1.2',
+        'XStatic-Bootstrap>=4.0.0',
         'XStatic-Font-Awesome>=6.2.1.0',
         'XStatic-CKEditor>=3.6.1.2',
         'XStatic-autosize',
         'XStatic-jQuery>=1.8.2',
-        'XStatic-jQuery-File-Upload>=10.31.0',
-        'XStatic-TWikiDraw-moin>=2004.10.23.2',
-        'XStatic-AnyWikiDraw>=0.14.2',
-        'XStatic-svg-edit-moin>=2012.11.15.1',
+        'XStatic-jQuery-File-Upload',
         'XStatic-JQuery.TableSorter>=2.14.5.1',
         'XStatic-Pygments>=1.6.0.1',
     ],
